tests/php/tests/includes/helper/test_class.wp-job-manager-helper-api.php

```diff
@@ -225,6 +225,24 @@ protected function default_invalid_response() {
 	}
 
 	protected function build_url( $args ) {
+		// The legacy endpoints are temporary. For now, translate `license_key` => `licence_key` at this point.
+		$args = $this->updateArrayKey( $args, 'license_key', 'licence_key' );
+
 		return 'https://wpjobmanager.com/?' . http_build_query( $args, '', '&' );
 	}
+
+	private function updateArrayKey( $array, $oldKey, $newKey ) {
+		if ( array_key_exists( $oldKey, $array ) ) {
+			$newArray = array();
+			foreach ( $array as $key => $value ) {
+				if ( $key === $oldKey ) {
+					$newArray[ $newKey ] = $value;
+				} else {
+					$newArray[ $key ] = $value;
+				}
+			}
+			return $newArray;
```

Comment:
You could have done something like:
```
if ( array_key_exists( $oldKey, $array ) ) {
  $array[$newKey] = $array[$oldKey];
  unset($array[$oldKey]);
}
return $array;
```
So we don't need to iterate over all the keys.

Reply:
Hey @aaronfc! Thank you for your review.
This implementation is because of the order of the keys. More details in this comment: https://github.com/Automattic/WP-Job-Manager/pull/2565#discussion_r1300776663

So I'm going ahead and merging it.